src/requirements/requirement-frontend-utils.ts

```diff
@@ -78,8 +78,6 @@ export function requirementAllowDoubleCounting(
 ): boolean {
   // All minor requirements are automatically double-countable.
   if (requirement.sourceType === 'Minor') return true;
-  // All grad program requirements are automatically not double-countable.
-  if (requirement.sourceType === 'Grad') return false;
```

Comment:
This is already implied for all requirements. Having an early `return false` means grad requirements cannot allow double counting, even with the `allowCourseDoubleCounting` flag.

Reply:
See the general comment

Reply:
See the general review.